src/lib/spinel/radio_spinel.hpp

```diff
@@ -695,6 +695,17 @@ class RadioSpinel : private Logger
      *
      */
     void SetDiagOutputCallback(otPlatDiagOutputCallback aCallback, void *aContext);
+
+    /**
+     * Gets the diag output callback.
+     *
+     * @param[output]  aCallback   A pointer to a function that is called on outputting diag messages.
+     * @param[output]  aContext    A pointer to the user context.
```

Comment:
Maybe  `[output]` also  work, but I think we use `[out]` generally.
```suggestion
     * @param[out]  aCallback   A pointer to a function that is called on outputting diag messages.
     * @param[out]  aContext    A pointer to the user context.
```